internal/gen/types/types.go

```diff
@@ -60,6 +60,9 @@ func Run(ctx context.Context, projectId string, dbConfig pgconn.Config, lang str
 			return err
 		}
 
+		// Extract version from image tag and trim 'v' prefix
+		postgrestVersion = strings.TrimPrefix(utils.Config.Api.Image, "postgrest/postgrest:v")
+
```

Comment:
Can we move this to its own check that also applies to remote database?

```suggestion
	if len(postgrestVersion) == 0 {
		// Extract version from image tag and trim 'v' prefix
		postgrestVersion = strings.TrimPrefix(utils.Config.Api.Image, "postgrest/postgrest:v")
	}
```

Reply:
It's also possible to connect to remote db via direct connection string, via --db-url flag. Although it's more common to hit our hosted api like you mentioned.

Another use case is connecting to self hosted db that's not on localhost.